From 8320d3b929f69a786174d14fc3a4d5021d136c4e Mon Sep 17 00:00:00 2001
From: Patrick Mooney <pmooney@pfmooney.com>
Date: Tue, 6 Jun 2017 23:49:26 +0000
Subject: [PATCH] OS-6156 die() misleads with KERNELBASE Reviewed by: Jerry
 Jelinek <jerry.jelinek@joyent.com> Reviewed by: Robert Mustacchi
 <rm@joyent.com> Approved by: Robert Mustacchi <rm@joyent.com>

---
 usr/src/uts/i86pc/os/trap.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/usr/src/uts/i86pc/os/trap.c b/usr/src/uts/i86pc/os/trap.c
index e84afc7903..192f3dbd32 100644
--- a/usr/src/uts/i86pc/os/trap.c
+++ b/usr/src/uts/i86pc/os/trap.c
@@ -32,7 +32,7 @@
 /*								*/
 
 /*
- * Copyright 2015 Joyent, Inc.
+ * Copyright 2017 Joyent, Inc.
  */
 
 #include <sys/types.h>
@@ -214,7 +214,7 @@ die(uint_t type, struct regs *rp, caddr_t addr, processorid_t cpuid)
 
 	curthread->t_panic_trap = &ti;
 
-	if (type == T_PGFLT && addr < (caddr_t)KERNELBASE) {
+	if (type == T_PGFLT && addr < (caddr_t)kernelbase) {
 		panic("BAD TRAP: type=%x (#%s %s) rp=%p addr=%p "
 		    "occurred in module \"%s\" due to %s",
 		    type, trap_mnemonic, trap_name, (void *)rp, (void *)addr,
-- 
2.21.0

